pages/alerts.rst

```diff
@@ -312,6 +315,9 @@ Configuration Color
 Custom Message
     The message that will be sent to Slack.  The data described above can be used in this template.
 
+Message Backlog Limit (optional)
+    Limit the number of backlog messages sent as part of the Slack notification.
```

Comment:
Should add something here to indicate that if the value is set to 0, no limit will be enforced (similar to the UI)

Reply:
I have added the line in sync with UI help block. ` Limit the number of backlog messages sent as part of the Slack notification. If set to 0, no limit will be enforced.`